src/js/components/Page/stories/CustomThemed/LeftColumn.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  Page,
+  Box,
+  Header,
+  Heading,
+  Paragraph,
+  Grid,
+  Card,
+  PageSection,
+  Grommet,
+} from 'grommet';
+
+const customTheme = {
+  page: {
+    customKind: {
+      alignSelf: 'start',
+      width: {
+        min: '200px',
+        max: '500px',
+      },
+      pad: {
+        small: 'medium',
+        medium: 'medium',
+        large: 'medium',
+      },
+      margin: { vertical: 'small', horizontal: 'small' },
+    },
+  },
+};
+
+export const LeftColumn = () => (
+  <Grommet theme={customTheme}>
+    <Box background="#35bd55">
+      <Heading>Global Header</Heading>
```

Comment:
Should this be a heading? or just plain text

Reply:
The stories in this PR aren't meant to be published to storybook, they were more for just testing. I'm working on creating storybook examples for page that we can actually publish. I will push these up soon.